app/controllers/users_controller.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def show
+  end
```

Comment:
Thought:

I've always found it strange to have an empty method, I believe our code analysis (Sonar) highlights this.

See: [Rails Convention Over Configuration in Action](https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-by-default-convention-over-configuration-in-action)

Your thoughts?

Reply:
i prefer explicit to fill in the magic myself! 

however, in this case this controller action needed amending anyway as it, and the view, were not doing what a restful route should do.

Changed it now